.github/workflows/ci.yml

```diff
@@ -97,3 +97,54 @@ jobs:
 
       - name: Run unit tests
         run: npm test
+
+  vrt-runner:
+    runs-on: ubuntu-latest-8-cores
+    strategy:
+      fail-fast: false
+      matrix:
+        shard: [1, 2]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: 'npm'
+      - name: Install dependencies
+        run: npm ci
+      - name: Build storybook
+        run: |
+          npm run build:new-tokens
+          cd docs/storybook
+          npm run build-storybook
+        working-directory: docs/storybook
+      - name: Install browsers
+        run: npx playwright install --with-deps
+      - name: Run storybook
+        id: storybook
+        run: |
+          npx serve -l 6006 storybook-static &
+          pid=$!
+          echo "pid=$pid" >> $GITHUB_OUTPUT
+          sleep 5
+        working-directory: docs/storybook
+      - name: Run VRT
+        run: npx playwright test --grep @vrt --shard="${{ matrix.shard }}/${{ strategy.job-total }}"
+      - name: Stop storybook
+        run: kill ${{ steps.storybook.outputs.pid }}
+      - name: Upload report
+        if: ${{ always() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: vrt-${{ matrix.shard }}
+          path: .playwright/report
+
+  vrt:
```

Comment:
Can we add a name here that is more descriptive? So that it does not say `CI / vrt-runner (1)`